.travis.yml

```diff
@@ -16,7 +17,7 @@ jobs:
   fast_finish: true
   include:
     - stage: Documentation
-      julia: 1.5
+      julia: 1.6
```

Comment:
You can leave it, but in reality, you do not need it. It's for documented only and Documenter can work just fine on old versions of Julia. Even more, you can run into some issues on new versions, but old ones is already battle-tested.